bee-crypto/src/ternary/sponge/batched_curlp/bct.rs

```diff
@@ -9,95 +9,119 @@
 // an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and limitations under the License.
 
-use std::slice::SliceIndex;
+use std::ops::{Deref, DerefMut, Range};
+
+#[derive(Clone, Copy, Debug)]
+pub struct BCTrit(pub usize, pub usize);
```

Comment:
Even if this is not reachable because not exported I'd use `pub(crate)` for clarity. But as you want, I don't really mind.